index.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" />
 </head>
+
 <body>
-    
+    <section class="city-name-container">
+        <h1>Weather Report</h1>
+        <h2>for the lovely city of...</h2>
+        <h2 id="newCity">Choose a city</h2>
```

Comment:
These should probably be a single `h2` (or potentially both be something else) since they don't both title different parts of the page.